pipeline/alignments/generate-space-alignment.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
FYI: When this is ready for review, especially since this is a new bash script, I'll probably request a `run_task` test for it. This is using `fast_align` and `atools` which may or not be available via a pip install. I wonder if this would be a good place for an initial docker test, or maybe just mock the binary out again.

Reply:
yes, I wanted to introduce docker tests for this but I think it's better to write tests as a follow-up since this PR is already quite big

Reply:
Also,  nothing new is happening in the alignments, we just skip sentence piece decoding. All the critical bugs will be caught on CI. Marian fails right away when the alignments mismatch the text.

Reply:
I added tests that use binaries from toolchain and run locally under docker or without it if you're on linux